### PR TITLE
added alert component

### PR DESCRIPTION
**Pull request by skiran252**

### CODE REVIEW

**Review by cjinc121 — APPROVED** [review]
looks good

File: styles/components/alert.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+.alert-warning {
+    background-color: #fff3cd;
+    color: #856404;
+    border-color: #ffeeba;
+}
+
+.alert-info {
+    background-color: #d1ecf1;
+    color: #0c5460;
+    border-color: #bee5eb;
```
Comment (amarphule) [inline styles/components/alert.css]: For colors, you can create variables and use them in CSS. Following this approach will help you to perform better while coding and you don't need to remember which hex color you used.
```
--background-color: #d1ecf1;
--primary-color: #0c5460;
--border-color: #bee5eb;
```

**Review by amarphule — CHANGES_REQUESTED** [review]
good work @skiran252  See the comment I made on the CSS file it will help in the future as well

**Review by amarphule — COMMENTED** [review]
good work @skiran252  See the comment I made on the CSS file it will help in the future as well

File: styles/components/alert.css (hunk 1 above)
Comment (skiran252) [reply styles/components/alert.css]: sure thanks this helps @amarphule should i create for as many colors as i have

File: docs/index.html [hunk 2]
```diff
@@ -51,17 +51,6 @@ <h3 class="sidebar-header">COMPONENTS</h3>
             <li class="docs-navbar-item" target="alert">
               <a href="#" target="alert">Alerts</a>
```
Comment (MaruthiPothuganti) [inline docs/index.html]: target allows only below values and also we don't need it here in the nav

image

File: docs/index.html [hunk 3]
```diff
@@ -51,17 +51,6 @@ <h3 class="sidebar-header">COMPONENTS</h3>
             <li class="docs-navbar-item" target="alert">
```
Comment (MaruthiPothuganti) [inline docs/index.html]: Here **target**  can be only used for anchor tags and doesn't have any purpose with **list** tags

**Review by MaruthiPothuganti — COMMENTED** [review]
Recommended changes: related to target attribute

File: docs/index.html (hunk 3 above)
Comment (skiran252) [reply docs/index.html]: please look at it now @MaruthiPothuganti

File: styles/components/alert.css (hunk 1 above)
Comment (amarphule) [reply styles/components/alert.css]: yes, you can create variables as many as you needed

**Pull request merged**